### PR TITLE
feat(performance): Add mobile landing page

### PR DESCRIPTION
**Pull request by Zylphrex**
This adds the first iteration of the mobile landing page to performance.

### CODE REVIEW

File: static/app/utils/discover/fields.tsx [hunk 1]
```diff
@@ -625,14 +625,21 @@ export enum WebVital {
...
+
+const MEASUREMENTS: Readonly<Record<WebVital | MobileVital, ColumnType>> = {
   [WebVital.FP]: 'duration',
   [WebVital.FCP]: 'duration',
   [WebVital.LCP]: 'duration',
   [WebVital.FID]: 'duration',
   [WebVital.CLS]: 'number',
   [WebVital.TTFB]: 'duration',
   [WebVital.RequestTime]: 'duration',
+  [MobileVital.AppStartCold]: 'duration',
```
Comment (k-fish) [inline static/app/utils/discover/fields.tsx:641]: Will this show up in the discover columns before it's EA'd?

File: static/app/views/performance/data.tsx [hunk 2]
```diff
@@ -441,6 +530,77 @@ function generateBackendPerformanceEventView(
   return eventView;
 }
 
+function generateMobilePerformanceEventView(
```
Comment (k-fish) [inline static/app/views/performance/data.tsx:533]: Not really a comment for this PR, just an aside; now that we have a quite a few of these event view generation functions we should probably DRY 🤔

File: static/app/views/performance/landing/content.tsx [hunk 3]
```diff
@@ -254,6 +292,7 @@ class LandingContent extends Component<Props, State> {
                 isActive={field === currentLandingDisplay.field}
               >
                 {label}
+                {beta && <FeatureBadge type="beta" noTooltip />}
```
Comment (k-fish) [inline static/app/views/performance/landing/content.tsx:295]: Alpha? 😄

File: static/app/views/performance/landing/utils.tsx [hunk 4]
```diff
@@ -49,6 +50,13 @@ export const LANDING_DISPLAYS = [
     label: 'Backend',
     field: LandingDisplayField.BACKEND,
   },
+  {
+    label: 'Mobile',
+    field: LandingDisplayField.MOBILE,
+    isEnabled: (organization: Organization) =>
```
Comment (k-fish) [inline static/app/views/performance/landing/utils.tsx:56]: I'd maybe rename this 'isShown' for clarity. I'd expect a UI element's 'enabled' to be css state usually (visible but grayed out).

**Review by k-fish — APPROVED** [review]
Nice work Tony 🎉

File: static/app/utils/discover/fields.tsx (hunk 1 above)
Comment (Zylphrex) [reply static/app/utils/discover/fields.tsx:641]: No it will not

File: static/app/views/performance/landing/content.tsx (hunk 3 above)
Comment (Zylphrex) [reply static/app/views/performance/landing/content.tsx]: changed in d79cabf

File: static/app/views/performance/landing/utils.tsx (hunk 4 above)
Comment (Zylphrex) [reply static/app/views/performance/landing/utils.tsx]: changed in d79cabf

**Pull request merged**